azurerm/resource_arm_app_service_plan.go

```diff
@@ -136,12 +139,16 @@ func resourceArmAppServicePlan() *schema.Resource {
 				ConflictsWith: []string{"properties.0.reserved"},
 			},
 
-			"maximum_number_of_workers": {
+			"maximum_elastic_worker_count": {
 				Type:     schema.TypeInt,
+				Optional: true,
 				Computed: true,
```

Comment:
What is the reasoning behind this being computed?

Reply:
Because by default this is 1 - so if the user doesn't set this value then it will be 1 coming back from the API and therefore will cause a diff